src/microparams-init.c

```diff
@@ -2409,6 +2409,58 @@ size_t xnn_init_f32_minmax_avxvnni_params(
 }
 #endif  // XNN_ARCH_X86 || XNN_ARCH_X86_64
 
+#if XNN_ARCH_X86 || XNN_ARCH_X86_64
+size_t xnn_init_s32_minmax_sse41_params(
+  union xnn_s32_minmax_params params[XNN_MIN_ELEMENTS(1)],
+  int32_t output_min,
+  int32_t output_max)
+{
+  for (uint32_t i = 0; i < 4; i++) {
+    params->sse41.min[i] = output_min;
+    params->sse41.max[i] = output_max;
+  }
+  return sizeof(params->sse41);
+}
+
+size_t xnn_init_s32_minmax_avx2_params(
+  union xnn_s32_minmax_params params[XNN_MIN_ELEMENTS(1)],
+  int32_t output_min,
+  int32_t output_max)
+{
+  for (uint32_t i = 0; i < 8; i++) {
+    params->avx2.min[i] = output_min;
+    params->avx2.max[i] = output_max;
+  }
+  // for (uint32_t i = 0; i < 7; i++) {
+  //   params->avx2.mask_table[i] = -1;
+  // }
+  // for (uint32_t i = 7; i < 14; i++) {
+  //   params->avx2.mask_table[i] = 0;
+  // }
+  return sizeof(params->avx2);
+}
+
+size_t xnn_init_s32_minmax_avx512f_params(
```

Comment:
Can avx512f use xnn_init_s32_minmax_scalar_params(?

Reply:
yes we are using xnn_init_s32_minmax_scalar_params